devicon.json

```diff
@@ -1923,6 +1923,36 @@
         "color": "#092e20",
         "aliases": []
     },
+    {
+        "name": "djangorest",
+        "altnames": [
+            "Django REST framework"
+        ],
+        "tags": [
+            "framework",
+            "rest",
+            "api",
+            "python",
+            "web"
+        ],
+        "versions": {
+            "svg": [
+                "original",
```

Comment:
The perfect version list doesn't exi-

Reply:
Sorry, but I don't get it.

Reply:
Just very nice to see all the versions there :)
So no complaints, just wanted to make a positive comment for once ;)

Reply:
Oh, I got the mood! 😅
I just got a little bit confused because I thought about `apache` icon.